docs/react-native/v2/guides/token-endpoint.mdx

```diff
@@ -3,11 +3,25 @@ title: Auth Token Endpoint Guide
 nav: 2.3
 ---
 
+## Overview
+
+100ms provides an option to get `App Tokens` without setting up a token generation backend service to simplify your integration journey while testing the [sample app](https://github.com/100mslive/100ms-web) or building integration with 100ms.
+
+You can find the token endpoint from the [developer page](https://dashboard.100ms.live/developer) in your 100ms dashboard.
+
+![Token endpoint](/guides/token-endpoint-dashboard.png)
+
+We recommend you move to your token generation service before you transition your app to production, as our token endpoint service will not scale in production.
+
+The `Client Apps` built using 100ms client SDKs require an `App Token` to join a room to initiate a video conferencing or live streaming session. Please check the [Authentication and Tokens guide](./../foundation/security-and-tokens)
```

Comment:
please call these "sample apps" like you have on the landing page - consistency is critical. and why is this in backticks?

Reply:
fixed it.